plugins/org.jkiss.dbeaver.ext.clickhouse/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -0,0 +1,13 @@
+# Translated By 2023 thenext600 (1556746401@QQ.com)
+#Properties file for org.jkiss.dbeaver.ext.clickhouse
+Bundle-Vendor = JKISS
```

Comment:
Please don't add it to property files with localization on Chinese as there is no translation. Could you remove this change?